internal/accumulated/run.go

```diff
@@ -342,6 +360,19 @@ func listenHttpUrl(s string) (net.Listener, bool, error) {
 	return l, secure, nil
 }
 
+// listenTCPorUnix will listen on a TCP or a Unix socket.
+// Using tmnet.ProtocolAndAddress to get the protocol and address path
+// works with either TCP or Unix listeners
+func listenTCPorUnix(s string) (net.Listener, error) {
+	protocol, address := tmnet.ProtocolAndAddress(s)
+	l, err := net.Listen(protocol, address)
+	if err != nil {
+		return nil, err
+	}
+
+	return l, nil
+}
+
```

Comment:
Instead of creating a new function, modify the existing one. `listenHttpUrl` is still an appropriate name, because it's still listening on HTTP, and we should still leave room for HTTPS. Instead of using `tmnet.ProtocolAndAddress`, add a case in `listenHttpUrl` for `u.Scheme == "unix"` and skip the path check if the scheme is unix.